style.css

```diff
@@ -364,3 +364,84 @@ button:disabled {
 .frameworks img {
   transform: rotate(45deg);
 }
+
+.first-article-contact {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.contact-para {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  width: 327px;
+  height: 216px;
+  color: #172b4d;
+  margin: 120px 0 73px;
+}
+
+#contact-form {
+  padding-bottom: 124px;
+  border-bottom: 1px solid #dfe1e6;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-bottom: 25px;
+}
+
+#contact-form input {
+  border-bottom: 1px solid #dbd8d7;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  width: 327px;
+}
```

Comment:
[OPTIONAL]
- [ ] I suggest you increase the height of the input fields a little so it's easier to tap on some mobile devices
- [ ] Also, kindly change to text color from the default of `#000` to that of the Figma design `#979493`